src/components/TopBar/CallButton.vue

```diff
@@ -151,6 +152,11 @@ export default {
 			return this.conversation.readOnly === CONVERSATION.STATE.READ_WRITE
 				&& this.participant.inCall !== PARTICIPANT.CALL_FLAG.DISCONNECTED
 		},
+		callButtonStyle() {
+			if (this.conversation.hasCall && !this.isBlockedByLobby) {
+				return { backgroundColor: `#5FDA34`, border: `1px solid #5FDA34` }
```

Comment:
`var(--color-success)`

Reply:
@skjnldsv  can we make var(--color-sucecss) a bit more vibrant? I think it's too dull, at least for this purpose...
![Screenshot from 2020-04-27 10-14-22](https://user-images.githubusercontent.com/26852655/80349748-fb794e80-886f-11ea-965a-b41c9a9fd00d.png)

Reply:
Sure! I think it's best to keep the same green on between our apps, but we can definitely improve the default green here! :+1:

Reply:
Ok!

Reply:
> @skjnldsv can we make var(--color-sucecss) a bit more vibrant? I think it's too dull, at least for this use case...
> ![Screenshot from 2020-04-27 10-14-22](https://user-images.githubusercontent.com/26852655/80349748-fb794e80-886f-11ea-965a-b41c9a9fd00d.png)

I was about to write that we need to make the green darker for better contrast with the text, and the --color-success does the job quite well. :) We can always tweak, but agree with keeping it consistent.